[2024-03-23T19:47:25.554+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_to_bigquery.load_data_to_BigQuery manual__2024-03-23T19:43:03.060824+00:00 [queued]>
[2024-03-23T19:47:25.571+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_to_bigquery.load_data_to_BigQuery manual__2024-03-23T19:43:03.060824+00:00 [queued]>
[2024-03-23T19:47:25.572+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 3
[2024-03-23T19:47:25.595+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_data_to_BigQuery> on 2024-03-23 19:43:03.060824+00:00
[2024-03-23T19:47:25.605+0000] {standard_task_runner.py:60} INFO - Started process 9226 to run task
[2024-03-23T19:47:25.609+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_to_bigquery', 'load_data_to_BigQuery', 'manual__2024-03-23T19:43:03.060824+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpif0jevmz']
[2024-03-23T19:47:25.613+0000] {standard_task_runner.py:88} INFO - Job 41: Subtask load_data_to_BigQuery
[2024-03-23T19:47:25.697+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_to_bigquery.load_data_to_BigQuery manual__2024-03-23T19:43:03.060824+00:00 [running]> on host 8a685a05c3f5
[2024-03-23T19:47:26.246+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhduc' AIRFLOW_CTX_DAG_ID='etl_to_bigquery' AIRFLOW_CTX_TASK_ID='load_data_to_BigQuery' AIRFLOW_CTX_EXECUTION_DATE='2024-03-23T19:43:03.060824+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-23T19:43:03.060824+00:00'
[2024-03-23T19:47:35.899+0000] {gbq.py:618} INFO - 541909 out of 541909 rows loaded.
[2024-03-23T19:47:35.900+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-23T19:47:35.918+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_to_bigquery, task_id=load_data_to_BigQuery, execution_date=20240323T194303, start_date=20240323T194725, end_date=20240323T194735
[2024-03-23T19:47:35.989+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-23T19:47:36.011+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
